### PR TITLE
fix(netlify): tests

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
updated tests to use uppercase methods

### CODE REVIEW

**Comment by alexanderniebuhr** [comment]
Current dependencies on/for this PR:
* main
  * **PR #96** <a href="https://app.graphite.dev/github/pr/withastro/adapters/96?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/withastro/adapters/96?utm_source=stack-comment).

**Comment by alexanderniebuhr** [comment]
@lilnasy don't think we need to release. It is only a change in the test suites. If users write their code with uppercase methods _(the correct way)_, it should work.

**Comment by lilnasy** [comment]
Didn't we have to exclude netlify from being released earlier? The latest version on NPM still has peerDependency issues.

File: .changeset/stale-boats-joke.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': minor
+---
+
+Fixes `@astrojs/netlify` to make it compatible with Astro 4.0.
```
Comment (lilnasy) [inline .changeset/stale-boats-joke.md:5]: the compatibility remains the same, right
```suggestion
Fixes an issue where this package could not be installed alongside Astro 4.0.
```

**Pull request merged**